Examples/GenericSerialization.cs

```diff
@@ -47,7 +47,7 @@ void Start()
             //Log.Debug("GenericSerialization", "main skill: {0}", uDefinedObject["name"]);
             //Log.Debug("GenericSerialization", "test: {0}", messageResponse);
 
-            var name = messageResponse.Context.Skills["main skill"]["user_defined"]["name"].ToString();
+            var name = messageResponse.Context.Skills.Skill.UserDefined["name"].ToString();
```

Comment:
does this work here?
```cs
messageResponse.Context.Skills.Skill.UserDefined.Get("name");
```

Reply:
Yep it worked, but I had to make this change: https://github.com/watson-developer-cloud/unity-sdk/pull/594/files#diff-e8323f26d84418a6500938a8e5996cc1, for it to work